src/actions/actions.ts

```diff
@@ -43,6 +43,42 @@ const compareKeypressSequence = function (one: string[], two: string[]): boolean
   return true;
 };
 
+const adjustVimSelectionPosition = function(vimState: VimState, start: Position, end: Position): any {
```

Comment:
Two comments here:

* `adjustVimSelectionPosition` is an ambiguous name. Something like `transformRangeForDeleteOperator` might be better. 
* Instead of `any` I would prefer using a TypeScript tuple type: `[Position, Position]`